kedro/config/common.py

```diff
@@ -82,10 +82,10 @@ def _get_config_from_patterns(
         if common_keys:
             sorted_keys = ", ".join(sorted(common_keys))
             msg = (
-                "Config from path '%s' will override the following "
-                "existing top-level config keys: %s"
+                "Config from path [not italic magenta]%s[/not italic magenta] will override the "
```

Comment:
```suggestion
                "Config from path [magenta]%s[/magenta] will override the "
```
I don't think you need to force it to be `not italic` here. Just comparing to the rich default styles, `repr.path` is not include `not italic` when others do.

Reply:
Thanks for the catch - I was following the convenition on  `repr.attrib_value` instead